### PR TITLE
Unify NativeBuildReport usage via jbang in GH workflows

### PR DESCRIPTION
**Pull request by famod**
Way simpler (IMO), with latest jbang version. Just like here: https://github.com/quarkusio/quarkus/blob/main/.github/workflows/jdk-early-access-build.yml#L109-L120

### CODE REVIEW

**Review by gastaldi — APPROVED** [review]
LGTM. Another way is to use the JBang GitHub  Action image, like in https://github.com/quarkusio/quarkus-extension-catalog/blob/main/.github/workflows/publish.yaml#L26

File: .github/workflows/native-cron-build.yml.disabled [hunk 1]
```diff
@@ -59,18 +59,14 @@ jobs:
...
         run: |
-          echo "The report step got status: ${STATUS}"
-          sudo apt-get update -o Dir::Etc::sourcelist="sources.list" \
-            -o Dir::Etc::sourceparts="-" -o APT::Get::List-Cleanup="0"
-          sudo apt-get install -y gnupg2 gnupg-agent
-          echo Installing SDKMAN
-          curl -s "https://get.sdkman.io" | bash
-          source ~/.sdkman/bin/sdkman-init.sh && \
-          sdk install jbang 0.21.0
           [[ ${JAVA_VERSION} = 8 ]] && ISSUE_NUMBER="6717" || ISSUE_NUMBER="6723"
```
Comment (gsmet) [inline .github/workflows/native-cron-build.yml.disabled:64]: Given we don't support JDK 8 anymore, we could simplify further, couldn't we?

Comment (famod) [reply .github/workflows/native-cron-build.yml.disabled:64]: I was a bit on the fence about this one but yeah, dropping this predicate makes sense.

**Comment by famod** [comment]
@gastaldi 
> Another way is to use the JBang GitHub Action image, like in https://github.com/quarkusio/quarkus-extension-catalog/blob/main/.github/workflows/publish.yaml#L26

That's what I first had for the JDK EA job but unfortunately, `/opt/hostedtoolscache` is not mounted into the container and so jbang (in the container) sees `JAVA_HOME` pointing to an unavailable path. This is very much an problem on GH's behalf, not specifically the jbang action's fault.
This only happens if the workflow uses one of those popular actions to set up a specific JDK. If you use the OOTB one then you're fine.

**Pull request merged**